openlibrary/plugins/openlibrary/js/edit.js

```diff
@@ -15,6 +15,63 @@ function update_len() {
     $('#excerpts-excerpt-len').html(2000 - len).css('color', color);
 }
 
+/**
+ * This is needed because jQuery has no forEach equivalent that works with jQuery elements instead of DOM elements
+ * @param selector - css selector used by jQuery
+ * @returns {*[]} - array of jQuery elements
+ */
+function getJqueryElements(selector){
+    const queryResult = $(selector);
+    return Array.from(queryResult).map((ele,index) => queryResult.eq(index));
+}
+
+export function initLanguageMultiInputAutocomplete() {
+    $(function() {
+        getJqueryElements('.language_multi_input_autocomplete').forEach(jqueryElement => {
+            jqueryElement.setup_multi_input_autocomplete(
+                'input.language-autocomplete',
+                // defined by jsdef
+                // eslint-disable-next-line no-undef
+                render_language_field,
+                {endpoint: '/languages/_autocomplete'},
+                {
+                    max: 6,
+                    // defined by jsdef
+                    // eslint-disable-next-line no-undef
```

Comment:
Rather than do this, could we define globals at the top of the page, so at a glance it's easier to see which ones are provided by the template?

/* global render_language_autocomplete_item, render_work_autocomplete_item */
/* Globals are provided by the __ template */

Reply:
This is much better. I'm not too familiar these sort of linter tools so I'm glad you do know.